src/gui/folderman.cpp

```diff
@@ -1531,13 +1531,23 @@ void FolderMan::restartApplication()
 
 Result<void, QString> FolderMan::unsupportedConfiguration(const QString &path) const
 {
-    if (numberOfSyncJournals(path) > 1) {
-        return tr("Multiple accounts are sharing the folder %1.\n"
-                  "This configuration is know to lead to dataloss and is no longer supported.\n"
-                  "Please consider removing this folder from the account and adding it again.")
-            .arg(path);
+    auto it = _unsupportedConfigurationError.find(path);
+    if (it == _unsupportedConfigurationError.end()) {
+        it = _unsupportedConfigurationError.insert(path, [&]() -> Result<void, QString> {
+            if (numberOfSyncJournals(path) > 1) {
+                const QString error = tr("Multiple accounts are sharing the folder %1.\n"
+                                         "This configuration is know to lead to dataloss and is no longer supported.\n"
+                                         "Please consider removing this folder from the account and adding it again.")
+                                          .arg(path);
+                qCWarning(lcFolderMan) << error;
+                if (Theme::instance()->warnOnMultipleDb()) {
+                    return error;
```

Comment:
Not sure I like this: In the logfile, you will find the Warning/Error, but in fact the code does not return the error. That will be very confusing in debugging situations. Rather add a hint that it was disabled for the log as well.